source/common/event/libevent_scheduler.h

```diff
@@ -108,6 +108,17 @@ class LibeventScheduler : public Scheduler, public CallbackScheduler {
   static void onPrepareForStats(evwatch*, const evwatch_prepare_cb_info* info, void* arg);
   static void onCheckForStats(evwatch*, const evwatch_check_cb_info*, void* arg);
 
+  static constexpr int flagsBasedOnEventType() {
+    if (Event::PlatformDefaultTriggerType == FileTriggerType::Level) {
```

Comment:
Will this compile with `if constexpr`?

Reply:
Yeah it compiles on msvc so I changed into it